gui/input_pane/input.py

```diff
@@ -1,22 +1,74 @@
+import pathlib
+
+from typing import Callable
+
 import PyQt5.QtWidgets as qtw
 
 class InputPane(qtw.QWidget):
 
-    def __init__(self):
+    def __init__(self, go_process: Callable):
         super().__init__()
-        self._input_label = qtw.QLabel("Input Filepath: ")
-        self._output_folder_label = qtw.QLabel("Output Folder: ")
+        self._input_file_label = qtw.QLabel("Input SVG: ")
+        self._output_file_label = qtw.QLabel("Trimmed SVG ")
+        
+        self._input_file_field = qtw.QLineEdit()
+        self._output_file_field = qtw.QLineEdit()
 
-        self._input_field = qtw.QLineEdit()
-        self._output_folder_field = qtw.QLineEdit()
-
-        self._button = qtw.QPushButton("Go")
+        self._in_button = qtw.QPushButton(
+                "Browse for Input", clicked=self._choose_input_file
+            )
+        self._save_location_button = qtw.QPushButton(
+                "Browse for Save", clicked=self._choose_out_file
+            )
+        self._go_button = qtw.QPushButton("Go", clicked=go_process)
 
         self._grid = qtw.QGridLayout()
 
-        self._grid.addWidget(self._input_label, 0, 0)
-        self._grid.addWidget(self._input_field, 0, 1)
-        self._grid.addWidget(self._output_folder_label, 1, 0)
-        self._grid.addWidget(self._output_folder_field, 1, 1)
-        self._grid.addWidget(self._button, 2, 0)
+        self._grid.addWidget(self._input_file_label, 0, 0)
```

Comment:
can you add more space for seeing the full path by default?

Reply:
its thicc